### DISCUSSION TITLE
How to fix security  `Application Error Disclosure` when building with nextjs

### DISCUSSION THREAD

**Discussion by quynhbui-teqnological** [vercel/next.js#30088] [Help]
Hi everybody.
My project is using Nextjs framework( **version:  "9.1.2**), it works very well. But, when I checked security to release production, I received a `risk level: Medium` about Application Error Disclosure. 

![image](https://user-images.githubusercontent.com/52997226/138042592-3580a448-eec4-49f1-8a96-6d0a6b1433a4.png)

Next, i checked to find the source code that make this. 
It is here 
```javascript
{e.exports={100:"Continue",101:"Switching Protocols",102:"Processing",200:"OK",201:"Created",202:"Accepted",203:"Non-Authoritative Information",204:"No Content",205:"Reset Content",206:"Partial Content",207:"Multi-Status",208:"Already Reported",226:"IM Used",300:"Multiple Choices",301:"Moved Permanently",302:"Found",303:"See Other",304:"Not Modified",305:"Use Proxy",307:"Temporary Redirect",308:"Permanent Redirect",400:"Bad Request",401:"Unauthorized",402:"Payment Required",403:"Forbidden",404:"Not Found",405:"Method Not Allowed",406:"Not Acceptable",407:"Proxy Authentication Required",408:"Request Timeout",409:"Conflict",410:"Gone",411:"Length Required",412:"Precondition Failed",413:"Payload Too Large",414:"URI Too Long",415:"Unsupported Media Type",416:"Range Not Satisfiable",417:"Expectation Failed",418:"I'm a teapot",421:"Misdirected Request",422:"Unprocessable Entity",423:"Locked",424:"Failed Dependency",425:"Unordered Collection",426:"Upgrade Required",428:"Precondition Required",429:"Too Many Requests",431:"Request Header Fields Too Large",451:"Unavailable For Legal Reasons",500:"Internal Server Error",501:"Not Implemented",502:"Bad Gateway",503:"Service Unavailable",504:"Gateway Timeout",505:"HTTP Version Not Supported",506:"Variant Also Negotiates",507:"Insufficient Storage",508:"Loop Detected",509:"Bandwidth Limit Exceeded",510:"Not Extended",511:"Network Authentication Required"}
```

It is a part in the core of nextjs and I need to improve it to continue to release production
![image](https://user-images.githubusercontent.com/52997226/138042999-d9afcb6f-950d-405a-9ac8-5d796c117eff.png)

How to remove this source code in the building process?

**Comment by gaston-flores** [comment]
That file seems like a list of HTTP status codes, hardly a security risk. The tool you are using here is probably searching for "Error" in the page, which matches to `500:"Internal Server Error"` in that file. 

I think the intention of that warning is to prevent showing an error to the user, which could potentially be used against you. If the source of the error is that list of status codes, I'd say this is a false positive that can be safely ignored.

**Reply by quynhbui-teqnological** [reply]
yes, it seem so, i will discuss with my team about the security.
Thank you so much for your reply
